data/mods/gen6megasrevisited/abilities.ts

```diff
@@ -284,6 +284,7 @@ export const Abilities: import('../../../sim/dex-abilities').ModdedAbilityDataTa
 		},
 		name: "Parental Bond",
 		rating: 4.5,
+		shortDesc: "This Pokemon's damaging moves hit twice. The second hit has its damage quartered.",
```

Comment:
you didn't actually edit the base power modification?

Reply:
Yeah, I initially went to do that but then someone reported that Parental Bond's second hit damage was already quartered, which confused me since I hadn't edited that part of its code since [I fixed a bug with it awhile ago](https://github.com/smogon/pokemon-showdown/pull/10507), but then I found that Parental Bond's second hit damage reduction is now coded in scripts and in combination with my unchanged code it gave the desired effect already, so I just left it alone and added the updated description.